src/pages/Sponsors.js

```diff
@@ -0,0 +1,50 @@
+import React from "react"
+import Kleinman from '../images/kleinman.png'
+import MT from '../images/M&T_Logo_Tagline.png'
+import SP2 from '../images/sp2logo.png'
+import WGY from '../images/Whartonglobalyouthprogram.png'
```

Comment:
make your naming scheme consistent.

Since you have so many images, might be worth it to split images into seperate subdirs i.e. /images/sponsors/overhypedpreprofessionaldualdegreeprogram.png